ci/audit.yml

```diff
@@ -0,0 +1,17 @@
+---
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: maven
+    tag: 3.5.0-jdk-8
+
+inputs:
+  - name: zebedee
```

Comment:
Not sure if it really matters - but I guess this should be `the-train` instead of zebedee?

Reply:
🤦

Reply:
FFS. 🤦